### PR TITLE
Remove slot machine challenges from Basic JS

### PR DESCRIPTION
**Pull request by erictleung**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (`Closes #XXXX`): Closes 
#### Description

Removes Slot Machine challenges as per discussion in #8062 because they were in beta but now deemed unnecessary.

Removed challenges:
- Create a JavaScript Slot Machine
- Add your JavaScript Slot Machine Slots
- Bring your JavaScript Slot Machine to Life
- Give your JavaScript Slot Machine some Stylish Images

I tried running `npm test` and these errors came up. I couldn't figure out how to get rid of them. The only thing I've removed are the challenges and made sure I didn't delete any bracket or parentheses.

```
    ✖ test exited without ending
    -----------------------------
      operator: fail


    ✖ plan != count
    ----------------
      operator: fail
      expected: 2
      actual:   3




  Failed Tests: There were 2 failures

    These challenges have no solutions

      ✖ test exited without ending
      ✖ plan != count


  total:     760
  passing:   758
  failing:   2
  duration:  8.6s
```

I've iteratively removed the challenges starting from "Create a JavaScript Slot Machine" and was able to remove the first three slot machine challenges with no problems. Removing the last one, "Give your JavaScript Slot Machine some Stylish Images," creates these errors.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -5327,710 +5327,6 @@
         "<h4>Instrucciones</h4>",
         "Usa <code>/\\S/g</code> para contar el número de caracteres que no están en blanco en <code>testString</code>."
       ]
-    },
```
Comment (Rafase282) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: I think the `},` should not be removed. Still checking.

Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: I thought that was it but I needed to remove the comma at the end. And the way GitHub highlights things makes it look like I've deleted it but the matching bracket is [here](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/8267/commits/efde04ea988317ea2c1a1ce86a26088b25662cb3#diff-5fbffcdf48f7976dd3b0216c387eebc2R5330) at the end of the file.

**Comment by erictleung** [comment]
Possibly distantly related to #5662 as it runs into the `plan != count` error.

**Comment by raisedadead** [comment]
@erictleung Closing as stale.  Please re-open a fresh PR.

**Pull request closed without merging**